sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsWithDownleveledStorage.js

```diff
@@ -0,0 +1,88 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to use the EventHubConsumerClient to process events from all partitions
+  of a consumer group in an Event Hubs instance, as well as checkpointing along the way.
+
+  This sample uses the `DownLevelContainerClient` to target an older version of the Storage Blob service.
```

Comment:
Change the language to be more generic in its application. Something like "This sample uses the "APISpecificStorageContainerClient" to use a specific version of the Storage Blob service."